README.md

```diff
@@ -18,19 +18,23 @@ Find out more about how to use the extension as a Dapp developper, cookbook, as
 
 Steps to build the extension and view your changes in a browser:
 
-1. Build via `yarn build`
-2. Install the extension
+1. Install supported Node.js version. Optionally via `nvm install`
+2. Install dependencies via `yarn`
+3. Build via `yarn build` or `yarn watch`
+4. Install the extension
   - Chrome:
     - go to `chrome://extensions/`
     - ensure you have the Development flag set
     - "Load unpacked" and point to `packages/extension/build`
     - if developing, after making changes - refresh the extension
   - Firefox:
-    - go to `about:debugging#addons`
-    - check "Enable add-on debugging"
-    - click on "Load Temporary Add-on" and point to `packages/extension/build/manifest.json`
+    - go to "This Firefox" at `about:debugging#addons`
```

Comment:
What does `"This Firefox"` mean?

Reply:
it's a menu item in latest Firefox 124.0.1 as shown below. if i go to `about:debugging#addons` it redirects me to `about:debugging#/runtime/this-firefox`

<img width="986" alt="Screenshot 2024-03-25 at 8 01 44 am" src="https://github.com/polkadot-js/extension/assets/6226175/a8193483-b658-4849-9055-974a82585e87">

Reply:
This is the default tab when going to `about:debugging#addons`, so I suggest to remove this mention as it may change.
```suggestion
    - go to `about:debugging#addons`
```